### ISSUE TITLE
algo/improvement: tracking losses for algorithms

### ISSUE THREAD

**Issue by sameerashahh** [DynaDojo/dynadojo#109]
**Is your request related to a problem? Please describe.**
When returning the training and/or validation loss, there is an inconsistency in naming the variables.

**Describe the improvement you are suggesting**
"train_loss", "val_loss" (if exists), should be the naming convention for all the algorithms when returning the losses dictionary. 
aug_ode, dmd, dnn, dnn_keras, lr, ode, need to be changed.
